_data-prepper/pipelines/configuration/sinks/opensearch.md

```diff
@@ -76,7 +76,9 @@ ism_policy_file | No | String | The absolute file path for an ISM (Index State M
 number_of_shards | No | Integer | The number of primary shards that an index should have on the destination OpenSearch server. This parameter is effective only when `template_file` is either explicitly provided in Sink configuration or built-in. If this parameter is set, it would override the value in index template file. For more information, see [Create index]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/).
 number_of_replicas | No | Integer | The number of replica shards each primary shard should have on the destination OpenSearch server. For example, if you have 4 primary shards and set number_of_replicas to 3, the index has 12 replica shards. This parameter is effective only when `template_file` is either explicitly provided in Sink configuration or built-in. If this parameter is set, it would override the value in index template file. For more information, see [Create index]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/).
 distribution_version | No | String | Indicates whether the sink backend version is Elasticsearch 6 or later. `es6` represents Elasticsearch 6. `default` represents the latest compatible backend version, such as Elasticsearch 7.x, OpenSearch 1.x, or OpenSearch 2.x. Default is `default`.
-enable_request_compression | No | Boolean | Whether to enable compression when sending requests to OpenSearch. When `distribution_version` is set to `es6`, default is `false`. For all other distribution versions, default is `true`.  
+enable_request_compression | No | Boolean | Whether to enable compression when sending requests to OpenSearch. When `distribution_version` is set to `es6`, default is `false`. For all other distribution versions, default is `true`. 
+serverless | No | Boolean | Determines whether the OpenSearch Backend is Amazon OpenSearch Serverless. When `true`, ISM policies are not supported in the Amazon OpenSearch Service, therefore any ISM-related configurations set in the sink will have no effect. Default is `false`.
+serverless_options | No | String | Additional configuration options when the backend of the `s3` sink is set to Amazon OpenSearch Serverless.
```

Comment:
The `serverless_options` configuration is a nested configuration. When configured, it will setup a network policy. It has the following fields:

* `network_policy_name` - The name of the network policy to create.
* `collection_name` - The name of the AOSS collection to configure
* `vpce_id` - A VPC endpoint to connect

Reply:
I'll add these options in a separate section.